lib/distribute.c

```diff
@@ -393,15 +446,15 @@ static int distribute_print(struct vty *vty, char *tab[], int is_prefix,
 	return has_print;
 }
 
-int config_show_distribute(struct vty *vty)
+int config_show_distribute(struct vty *vty, const char *vrfname)
```

Comment:
please add a `distribute_ctx` parameter in the beginning and remove the `vrfname` parameter